$OpenBSD: patch-mk_config_mk_in,v 1.6 2011/04/11 14:30:33 kili Exp $

Don't build shared libs for now.

Remove now unused LLVM configuration setting.

--- mk/config.mk.in.orig	Sat Mar 26 19:10:06 2011
+++ mk/config.mk.in	Tue Mar 29 19:26:21 2011
@@ -107,7 +107,6 @@ GhcProfiled=NO
 PlatformSupportsSharedLibs = $(if $(filter $(TARGETPLATFORM),\
 	i386-unknown-linux x86_64-unknown-linux \
 	i386-unknown-freebsd x86_64-unknown-freebsd \
-	i386-unknown-openbsd x86_64-unknown-openbsd \
 	i386-unknown-mingw32 \
 	i386-unknown-solaris2 \
 	i386-apple-darwin powerpc-apple-darwin),YES,NO)
@@ -145,9 +144,6 @@ OsSupportsNCG=$(strip $(patsubst $(HostOS_CPP), YES, $
 GhcWithNativeCodeGen=$(strip\
     $(if $(filter YESYESNO,\
 		  $(OsSupportsNCG)$(ArchSupportsNCG)$(GhcUnregisterised)),YES,NO))
-
-# Build a compiler with the llvm code generator backend
-GhcWithLlvmCodeGen=NO
 
 HaveLibDL = @HaveLibDL@
 
